Software/vortex_ws/src/motion_planning/README.md

```diff
@@ -1 +1,192 @@
-Motion planning package
\ No newline at end of file
+# motion planning and control module package
+
+## description
+
+this package represents four modules (localization,global planner, local planner and motion control )
+
+Localization       : determine AUV's place in the 3d space around it and also determines the location of the obstacles and objects surrounding it.
+
+The Global Planner : give the AUV the ability to plan the correct and feasible path through the starting point to its goal.
+The Local Planner  :generate a set of control commands that‏ moves the AUV on the path generated by the Global Planner but with taking the dynamics into consideration while also trying to avoid obstacles.
+
+motion control :translate the set of control commands from the Local Planner into actions for the thrusters‏
+
+### License
+
+The source code is released under a [GNU GENERAL PUBLIC LICENSE].
+
+**Author: vortex-co<br />
+Affiliation: [VorteX-Co](https://vortex-co.com/home)<br />
+Maintainer: vortex-co, info@vortex-co.com**
+
+The Vision module package has been tested under [ROS2] eloquent on Ubuntu 18.04 
+
+## Table of contents
+* [Installation](#Installation)
+* [Usage](#Usage)
+* [Config files](#Config-files)
+* [Launch files](#Launch-files)
+* [Hardware](#hardware)
+
+
+## Installation
+### Installation from Packages
+
+To install all packages from the this repository as Debian packages use
+
+    sudo apt-get install ros-noetic-...
+    
+Or better, use `rosdep`:
+
+	sudo rosdep install --from-paths src
+
+### Building from Source
+
+#### Dependencies
+
+- [Robot Operating System (ROS)](https://index.ros.org/doc/ros2/) (middleware for robotics),
+
+
+#### Building
+
+To build from source, clone the latest version from this repository into your  workspace and compile the package using
+
+	cd workspace/src
+	git clone https://github.com/VorteX-co/VAUV/tree/master/Software/vortex_ws/src/vision
+	cd ../
+	rosdep install --from-paths . --ignore-src
+	catkin_make
+
```

Comment:
also remove this and we are not building using catkin, please do not copy and paste without reading we are using colcon but anyway also remove this part